class.jetpack-plan.php

```diff
@@ -153,16 +298,9 @@ public static function get() {
 		}
 
 		// Define what paid modules are supported by professional plans.
```

Comment:
Should this comment say "business"?

Reply:
I'm not sure, but looking at the code that follows, I would say so, yes.

Reply:
Professional and Business are the same thing. On WP.com, it is called "Business", in Jetpack it is called "Professional".